kinto/core/resource/__init__.py

```diff
@@ -962,6 +962,11 @@ def _extract_filters(self):
                 "description": f"Invalid value for {param}",
             }
 
+            # Return 400 if _limit is not a string
+            if "like" in param:
+                if not isinstance(value, str):
+                    raise_invalid(self.request, **error_details)
+
```

Comment:
If you move this down a little bit, you can use the parsed `operator` to check whether it is exactly a `like` query (and not e.g. a check for a field like `likes_pizza`).

Reply:
That  `like in param` could be troublesome. Fixed it now to use `operator` instead! Thanks!